### PR TITLE
Add ESLint #13

### PR DESCRIPTION
**Pull request by beatrizsmerino**
Install packages npm to ESLINT AIRBNB and created npm script 'lint'

### CODE REVIEW

File: lib/check.js [hunk 1]
```diff
@@ -1,92 +1,103 @@
...
     response = await axios.get(url);
   } catch (error) {
-    if (error.response) return response = error.response;
-    if (error.code) return response = { status: error.code }
+    if (error.response) {
+ return response = error.response;
+}
+    if (error.code) {
+ return response = {status: error.code};
+}
```
Comment (inacior) [inline lib/check.js:19]: I this this is indentation error

File: lib/check.js [hunk 2]
```diff
@@ -1,92 +1,103 @@
...
+};
 
-const _getProjects = async ({ file, env }) => {
+const _getProjects = async({file, env}) => {
   const fileExists = await fs.pathExists(file);
 
-  if (!fileExists) throw 'INVALID_JSON';
+  if (!fileExists) {
+ throw 'INVALID_JSON';
+}
```
Comment (inacior) [inline lib/check.js:40]: same.

File: lib/check.js [hunk 3]
```diff
@@ -1,92 +1,103 @@
...
 
   const allProjects = await fs.readJson(file);
 
-  const projects =  allProjects[env];
+  const projects = allProjects[env];
 
-  if (!projects) throw 'INVALID_ENV';
+  if (!projects) {
+ throw 'INVALID_ENV';
+}
```
Comment (inacior) [inline lib/check.js:48]: same.

File: lib/check.js [hunk 4]
```diff
@@ -1,92 +1,103 @@
...
+    return _catchGlobalError({file, error});
   }
 
-  const callChain = Object.entries(projects).map(async ([ name, options]) => {
-    spinnies.add(name, { text: name, color: 'gray' });
+  const callChain = Object.entries(projects).map(async([
+name,
+options
+]) => {
+    spinnies.add(name, {text: name, color: 'gray'});
```
Comment (inacior) [inline lib/check.js:77]: same.

**Review by inacior — CHANGES_REQUESTED** [review]
Your PR have some indentation errors, can you fix this?

File: package.json [hunk 5]
```diff
@@ -8,7 +8,8 @@
   "scripts": {
     "build": "webpack --config webpack.config.js",
     "test": "jest",
-    "test:coverage": "jest --coverage"
+    "test:coverage": "jest --coverage",
+    "lint": "eslint 'lib/**/*.js' --fix"
```
Comment (inacior) [inline package.json:12]:
```suggestion
    "lint": "eslint 'lib/**/*.js'",
    "lint:fix": "eslint 'lib/**/*.js --fix'"
```

Comment (beatrizsmerino) [reply package.json:12]: Yes, in a few hours I will reconnect and I will

**Comment by beatrizsmerino** [comment]
Great I find the problem.
The Check **Node.js CI / build (10.x) (pull_request)** failed. It has this error:
**SyntaxError: /home/runner/work/is-my-env-alive/is-my-env-alive/tests/lib/check.spec.js: Support for the experimental syntax 'classProperties' isn't currently enabled**

I install this package: `@babel/plugin-proposal-class-properties`
And use it on the `webpack.config.js`, `babel.config.js` and `.eslintrc`
Issue Reference: [https://github.com/babel/babel/issues/10095](https://github.com/babel/babel/issues/10095)

File: lib/check.js (hunk 1 above)
Comment (beatrizsmerino) [reply lib/check.js]: Now it is Ok

**Comment by beatrizsmerino** [comment]
@inacior It is fixed

**Comment by inacior** [comment]
Hi! I will review your code at night, ok? :)

File: package.json [hunk 6]
```diff
@@ -1,39 +1,49 @@
...
+	"version": "1.1.3",
+	"description": "A package thats help you to check if your environment projects is alive!",
+	"main": "dist/check.js",
+	"bin": "./dist/command.js",
+	"preferGlobal": true,
+	"scripts": {
+		"build": "webpack --config webpack.config.js",
+		"test": "jest",
+		"test:coverage": "jest --coverage",
+		"lint": "eslint 'lib/**/*.js' --fix"
```
Comment (inacior) [inline package.json:12]:
```suggestion
		"lint": "eslint 'lib/**/*.js'",
		"lint:fix": "eslint 'lib/**/*.js' --fix"
```

**Review by inacior — CHANGES_REQUESTED** [review]

**Comment by inacior** [comment]
With merge of #15 PR, you file is with some conflicts, can you fix it?

**Comment by beatrizsmerino** [comment]
Thanks!! I fix the conflicts

**Comment by inacior** [comment]
Thanks a lot, @beatrizsmerino !

**Pull request merged**